bundles/core/src/main/java/com/adobe/cq/wcm/core/components/util/ComponentUtils.java

```diff
@@ -49,6 +59,38 @@
      * Length of the ID hash.
      */
     private static final int ID_HASH_LENGTH = 10;
+    
+    /**
+     * Attribute on JCR node that captures component data 
+     */
+	public static final String CQ_STYLEIDS = "cq:styleIds";
+
+	 /**
+     *  Node under content policy that captures different styles
+     */
+	public static final String CQ_STYLES = "cq:styles";
+
+	
+	/**
+     * Node under content policy that captures different style groups
+     */
+	public static final String CQ_STYLEGROUPS = "cq:styleGroups";
+
+	
+	 /**
+     * Child nodes that contain each style definition under a specific group
+     */
+	public static final String CQ_STYLEID = "cq:styleId";
+
+	 /**
+     * Equivalent CSS classes for styleIds
+     */
+	public static final String CQ_STYLECLASSES = "cq:styleClasses";
+	
+	 /**
+     * Default CSS class to provide in absence of any selection
+     */
+	public static final String CQ_STYLECLASSES_DEFAULT = "cq:styleDefaultClasses";
```

Comment:
We should use the constants already defined in `com.adobe.cq.wcm.style.StyleConstants` instead of redefining them.

Reply:
Wasn't able to locate the StyleConstants class using project dependencies. If you can please share the reference URL to the code you are referring to, I shall make the required changes. Thank you !

Reply:
Both `com.adobe.cq.wcm.style.StyleConstants` and `ComponentStyleInfo` are exposed by the `com.adobe.cq.com.adobe.cq.wcm.style` bundle as internal AEM APIs. Perhaps it makes sense to make those public so they could be leveraged here. I'll start a conversation with the rest of the development team.

Reply:
@vladbailescu - Does this mean, the changes shall appear in next version of uber jar? Also any timeline on this? Just wanted to set expectation to my team on leveraging these changes so they don't end up creating new proxy components and sling models to support the capabilities

Reply:
@friendlymahi, the package should now be available in the AEM SDK: https://javadoc.io/doc/com.adobe.aem/aem-sdk-api/latest/com/adobe/cq/wcm/style/package-summary.html (since `2021.3.5026.20210309T210727Z-210225`)

Reply:
It should also be available in the 6.5.8 uber-jar (though https://javadoc.io/doc/com.adobe.aem/uber-jar/latest/index.html is not updated yet)

Reply:
The usage pattern is `resource.adaptTo(ComponentStyleInfo.class)`

Reply:
@vladbailescu - Made the changes per your recommendation. However I did come across some issues and I had to take few extra steps that may require your review.

- Updating uber jar version broke some existing classes as the corresponding interfaces seem to be having new methods that require impl in child classes. So added placeholder methods in EnhancedRendition.java and MockHtmlLibraryManager.java
- SlingBindings didn't work in Junit. So updated Sling API version in POM file (was already targeting only scope=test) with v2.22.0 (matching with what we have in AEM v6.5.8)
- After these changes, for some reason OSGI services injection fail within my Junit. Unsure if the same can happen in pipeline aswell.

Please advise on the next steps. Thank you !